backend/tracim_backend/models/context_models.py

```diff
@@ -87,7 +87,7 @@ def __init__(
         auth_types: List[str],
         saml_idp_list: List[Dict[str, str]],
         user__read_only_fields: Dict[str, List[str]],
-        app_custom_actions: List[Dict[str, str]],
+        app_custom_actions: Dict[str, str],
```

Comment:
Set type as `Dict[str, Any]`

Reply:
I think the custom actions type is `Dict[str, Dict]`, no?

Reply:
According to JSON specification it is a `Dict[str, Any]` (could be a Dict, a bool, a string, a float, a list, and an int)

Now we are sure that it is atleast a `Dict[str, List[Any]]` but since we're expecting a JSON, i'd prefer to stick to JSON's "type definition"

Reply:
I read again the issue. The spec is `Dict[str, List[Dict[Str, Any]]]`.

So the type should by `Dict[str, List]`, no ?

The strategy behind the issue is: "for some clearly identified nodes, we can add a list of extra actions (dynamically created through a set of rules)". So Str -> List makes sense.

Reply:
We can go for `Dict[str, List[Dict[Str, Any]]]`

Reply:
:sweat_smile:

Reply:
will fix

Reply:
fixed